src/ray/ray_config.h

```diff
@@ -228,6 +230,7 @@ class RayConfig {
         object_manager_repeated_push_delay_ms_(60000),
         object_manager_default_chunk_size_(1000000),
         num_workers_per_process_(1),
+        inline_object_max_size_bytes_(512),
```

Comment:
We also have to add the new field to this [method](https://github.com/ray-project/ray/blob/master/src/ray/ray_config.h#L112), which allows config options to be set at runtime.